vars/sconsBuild.groovy

```diff
@@ -251,5 +295,26 @@ def call(Map config = [:]) {
     if (!config['returnStatus'] && (rc != 0)) {
       error "sconsBuild failed for ${full_script}"
     }
+    if (config['test_files']) {
```

Comment:
```suggestion
    if (config['stash_files']) {
```

Maybe a bit more self-explanatory when reading the calling side of this function?

Reply:
Yes.  I will have to change the DAOS PR-3104 to match.